catlas/config_validation.py

```diff
@@ -180,23 +185,57 @@ def validate_folder_exists(field, value, error):
         "required": False,
         "type": "list",
         "schema": {
-            "type": "dict",
-            "schema": {
-                "checkpoint_path": {
-                    "type": "string",
-                    "check_with": validate_path_exists,
-                    "regex": ".*.pt",  # cerberus doesn't understand re "$"; requires full match
+            "anyof": [
+                {
+                    "type": "dict",
+                    "schema": {
+                        "checkpoint_path": {
+                            "type": "string",
+                            "check_with": validate_path_exists,
+                            "regex": ".*.pt",  # cerberus doesn't understand re "$"; requires full match
+                        },
+                        "gpu": {"type": "boolean", "required": True},
+                        "label": {
+                            "required": True,
+                            "type": "string",
+                        },
+                        "number_steps": {
+                            "type": "integer",
+                        },
+                        "batch_size": {"type": "integer"},
+                        "gpu_mem_per_sample": {"type": "float"},
```

Comment:
Randomly, I think this means you won't be able to specify "None"- you have to leave it blank or use a float.  Bit unintuitive but shouldn't be an issue

Reply:
Got it - I think this should be ok. People can comment out the line if they want.